src/main/scala/autoversion/conventional/ConventionalCommits.scala

```diff
@@ -0,0 +1,19 @@
+package autoversion.conventional
+
+import autoversion.AutoVersionPlugin
+import autoversion.Keys.{bugfixRegexes, majorRegexes, minorRegexes}
+import sbt.{AutoPlugin, Setting}
+
+/**
+ * An additional opt-in plugin that enables Conventional Commit style patterns for bumping the version
+ */
+object ConventionalCommits extends AutoPlugin {
+  override def requires = AutoVersionPlugin
+
+  override def projectSettings: Seq[Setting[_]] =
+    Seq(
+      majorRegexes := Seq(""".*BREAKING[-\s]CHANGE: .*""".r, "^(.*!: ).*".r),
+      minorRegexes := Seq("^feat.*: .*".r),
+      bugfixRegexes := Seq("^fix.*: .*".r),
```

Comment:
This overrides the autoversion defaults doesn't it? Since those settings are Sequences of regular expressions, maybe this plugin should put its own first and append the inherited value from autoversion?

Reply:
It does, and doesn't really give the option of also keeping the old patterns.  Additive patterns rather than alternative is a good suggestion, will update.

Reply:
Addressed in the latest commit